### DISCUSSION TITLE
怎么手动安装language· server ,  ccls 和 clangd  已经不能 用 LspInstallInfo导入了

### DISCUSSION THREAD

**Discussion by Dingjinjin-chihiro** [nshen/learn-neovim-lua#121] [Q&A]
![image](https://user-images.githubusercontent.com/81962954/193441226-cdb92a84-5d2d-4b6f-af54-625b185bcbd9.png)
难受啊啊啊啊啊，主力语言是C/C++ 后端

**Comment by Dingjinjin-chihiro** [comment]
已经解决·

**Comment by Engin-Hong** [comment]
请问安装到哪个位置?
